scripts/ppt_utils.py

```
@@ -0,0 +1,189 @@
+def _emu_to_px(emu_value) -> int:
+    if emu_value is None:
+        return 0
+    return int(emu_value / _EMU_PER_PX)
+
+
+def _try_font(size: int) -> ImageFont.FreeTypeFont:
+    for name in ("arial.ttf", "DejaVuSans.ttf", "Helvetica.ttf"):
+        try:
+            return ImageFont.truetype(name, size)
+        except (OSError, IOError):
+            continue
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)